src/HW_models/NRF_RTC.h

```diff
@@ -21,6 +21,7 @@ void nrf_rtc_update_COUNTER(int rtc);
 void nrf_rtc_regw_sideeffect_TASKS_START(int i);
 void nrf_rtc_regw_sideeffect_TASKS_STOP(int i);
 void nrf_rtc_regw_sideeffect_TASKS_CLEAR(int i);
+void nrf_rtc_regw_sideeffect_TASKS_TRIGGER_OVERFLOW(int i);
```

Comment:
Is there a reason for not using the original register name here (i.e. `TASKS_TRIGOVRFLW`) and not keeping this function name consistent with the others?

Reply:
@ankuns can you answer this review comment?

Reply:
I don't remember the reason for such decision, I will change it to TASKS_TRIGOVRFLW

Reply:
Then please update also names of `nrf_rtc*_TASKS_TRIGGER_OVERFLOW` functions.

Reply:
Ok, I know - the value representing the task in the `nrf_rtc_task_t` is `NRF_RTC_TASK_TRIGGER_OVERFLOW` Hence it wasn't clear at first which naming convention to use. However probably in this case TASKS_TRIGOVRFLW is better

Reply:
Changes should be already on repo.